### PR TITLE
Style leaderboard

### PR DESCRIPTION
**Pull request by kenwilde1**
* Restyled Leaderboard, added headings and Gold, Silver & Bronze icons
* Fixed firebase issue with being called twice
* Cleaned up unneeded code in `main.js`

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -76,15 +76,34 @@ ol {
...
+  display: flex;
+  justify-content: space-between;
+}
+
 .leaderboard-item {
   border-bottom: 1px solid var(--teritary-color);
 }
 
+#gold-medal {
+  color: goldenrod;
```
Comment (auxfuse) [inline assets/css/style.css]: Hi Kenny, can the color keywords be converted to hexadecimal values please? It keeps it consistent with other color calls throughout the file and also the root variables defined.

**Review by auxfuse — COMMENTED** [review]
@kenwilde1 one comment for you to look at 😉 Once that's done I think we can go ahead and merge 🤜
Thanks for getting this in so quick!

File: assets/css/style.css (hunk 1 above)
Comment (kgwilde) [reply assets/css/style.css]: Named colours converted to hex codes in latest commit

**Review by auxfuse — APPROVED** [review]
LGTM! 😉🍀

**Pull request merged**